source/routing/raptor_solution_reader.cpp

```diff
@@ -600,4 +473,95 @@ std::vector<Path> read_solutions(const RAPTOR& raptor,
     }
 }
 
+Path make_path(const Journey& journey, const type::Data& data) {
+    Path path;
+    const auto posix = [&data](DateTime dt) { return to_posix_time(dt, data); };
+
+    path.nb_changes = journey.sections.size() - 1;
```

Comment:
journey.sections do not have stay in, the sections correspond to the pt layers in raptor.